services/worker-manager/schemas/v1/config-aws.yml

```diff
@@ -13,7 +13,9 @@ properties:
         region:
```

Comment:
I think region will need to move into `launchConfigs`? Or perhaps there will be a new array above this that includes both region and launch configs